pkg/cmd/init.go

```diff
@@ -408,6 +408,60 @@ func initKubeconfigs(
 		klog.Warningf("Unable to remove stale kubeconfigs: %v", err)
```

Comment:
Just realized, do we cleanup stale kubeconfigs for NamedCertificates no longer present in the config?

Reply:
yes its taken cared before generating the kubeconfig by running cleanupStaleKubeconfigs, https://github.com/openshift/microshift/blob/main/pkg/cmd/init.go#L407